JS/js-unit-testing_math-functions/index.test.js

```diff
@@ -0,0 +1,59 @@
+import { add } from "./index.js";
```

Comment:
you can import more functions from the same file in one line by having them all in {} seperated with comma. e.g. :  import { add , subtract } from "./index.js"